dts/bindings/misc/nxp,rtxxx-adsp-ctl.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+include: [base.yaml]
+
+compatible: "nxp,rtxxx-adsp-ctl"
+description: NXP i.MX RTxxx DSP control driver
+
+properties:
+  "#address-cells":
+    const: 1
+  "#size-cells":
+    const: 1
+  sysctl:
+    required: true
+    type: phandle
+    description: phandle to the SYSCTL (most likely clkctl0) node
```

Comment:
Why does the comment here state "most likely clkctl0" ?

Reply:
@danieldegrasse That is an artifact of a thought process that was wrong. Will remove.